script/generateAlmostSketch.js

```diff
@@ -100,7 +102,10 @@ export function snapshotSymbols({ suffix = '' }) {
 
     layers
       .filter(layer => layer !== null)
-      .forEach(layer => symbol.addLayer(layer));
+      .forEach(layer => {
+        symbolMiddleware({layer, SVG, Text, RESIZING_CONSTRAINTS});
```

Comment:
Is there a reason we only expose the `SVG` and `Text` classes? Would other html2asketch classes not be useful here, for completeness?

Reply:
if you look at the demo in the pr description it lets you check the type of layer you're working with so you can change the resizing options.

```js
if(layer instanceof SVG) {
  layer.setResizingConstraint(TOP, LEFT, WIDTH, HEIGHT);
} else {
  layer.setResizingConstraint(LEFT, TOP);
}
```

I'll pass through `ShapeGroup` & `Rectangle` classes too.

Reply:
Yep, was just concerned this might only be exposing what your particular use case, requires, rather than being a general purpose middleware.